### ISSUE TITLE
Replace "<planet>" with "<planet>, <system>" in mission titles to make game friendlier to newbies

### ISSUE THREAD

**Issue by Tadrix** [endless-sky/endless-sky#2229]
I admit, when I was new to Endless Sky, mission screen wasn't especially handy. The map part was good, but I couldn't associate entries in available mission list with blips on the map without clicking on them, because they all mentioned planets. "Colonists to Longjump" — there's no Longjump in the map, even if system is explored.

Also, I suspect that in the future with so many planets they'll be referred as addresses are today. On snail mail envelope we write _city_, _country_, as in "Berlin, Germany", so it seems perfectly logical to someone on Rust to write "Prime, Betelgeuse".

I propose to create a text replacement <­address­>, which would be "<­planet­>, <­system­>" and replace all <­planet­>s in job mission names with it. Or just replace <­planet­> with <­planet­>, <­system­> in visible names (this I could do myself). Keeps names short, makes things simpler, trains to recognize planets.

**Comment by KiLEdEnNis** [comment]
currently the missions have <destination> which is the planet (if I remember correctly, on my laptop so hard to check), so <planet>, <system> wouldn't work (I think)

and I personally wouldn't like names like "Delivery to <planet>, <system>", I'd rather have "Delivery to <planet> in the <system> system" which would be too long, but either way that'd be an improvement that I think should be made for newbies

but we'd also need to widen the job panel in the UI because most names wouldn't fit: "Passenger transport to New Kansas, Muhlifain" for example (just picked a longer one from my current job board)

in short: should be done, needs job board widening to pair it with

**Comment by Bladewood** [comment]
@KiLEdEnNis < and > characters tend to behave poorly when you surround a word with them.

**Comment by KiLEdEnNis** [comment]
oh ye imma fix that, ty :P

**Comment by Amazinite** [comment]
Just read the description of the job, which has the planet and system. The job names only have the planet because having the planet and the system is too long.

**Comment by Tadrix** [comment]
@Amazinite I realize you've been here a long time and remember everything, and I'm starting to remember which planets are were too at this point, but the job board list, as it is now, isn't really transparent or helpful. You said it yourself — read job description. For every damn job on every damn planet along the way. It would be much better if one could look at list and see — oh, these are two destinations in the same system.

@KiLEdEnNis Wiki [says](https://github.com/endless-sky/endless-sky/wiki/CreatingMissions#text-replacements) that there are text replacements:
<­planet­> = destination planet
<­system­> = destination system
<­destination­> = "<­planet­> in the <­system­> system"
We just need a short version for destination to fit mission panel, which is what I proposed.
This is, of course, if we keep job ads as newspaper ads.

Other approach for simple jobs would be to make a table, something similar to what I suggested in #494 
Mission name, cargo, passengers, deadline (if any), waypoints (if any), destination system, destination planet, payment. But I myself think that job board is fine as it is, if only we'd show not only destination planet but also in which system it is in the mission name.

**Comment by Amazinite** [comment]
"Large rush delivery to Carbuncle Station" nearly runs off the UI panel as is. "Large rush delivery to Carbuncle Station, Elinath" is probably going to run off.

**Comment by Tadrix** [comment]
Probably. Are there possibilities to make these UI panels some 30 pixels wider?

**Comment by MessyMix** [comment]
@Tadrix that would be very unfriendly to smaller monitors, and the game should be as accessible as possible.

**Comment by Tadrix** [comment]
How small are we talking about? For reference, I'm still using 1280×1024 (which for today's standards is pretty small for a tabletop PC). The map view so far is bearable.

**Comment by jafdy** [comment]
I'm divided on this. 
+ Lets you check the size and destination system of the job at the same time.
+ Reduces confusion for new players
- It's bigger.
- Coding time, I don't think this is high priority.
~ Most systems have the only 1 land-able planet so this is a bit unneeded. 
~ You could just write "<­planet­>, <­system­>"

**Comment by Tadrix** [comment]
Should I rewrite jobs.txt to have "<­planet­>, <­system­>" in mission names, or UI panel size means that now this would be a bad idea?

**Comment by endless-sky** [comment]
This really doesn't seem necessary to me.

If you click on any mission (or select it, using the arrow keys) the map highlights its destination. And besides, it's much more efficient to do things the other way around - clicking on the system to cycle through the missions available there.

**Comment by Tadrix** [comment]
Agreed, but the problem is, some systems have multiple destinations, and there's no way to sort or identify them other than to cycle through missions… and the game will randomly include the missions already taken, which makes the task harder. For example, if I'm doing a timed mission and can afford one landing on the way, I naturally want to know which missions pay best and end in the same destination, not the same system. Right now the only way to do this is either writing down or committing to memory these things until cycled through all of them. Every time. Soon enough it becomes exhausting.

Then again, maybe there's a problem with my playing style, but I thought ES try to accommodate different approaches (and even balance them).

**Comment by Pointedstick** [comment]
Seems like there isn't a lot of desire to change this. It's just one of those little things you need to learn, but I don't think it's all that difficult. And as MZ says, in a lot of ways it's better to choose the region of space you want to fly towards and cycle through the missions there by clicking on the systems.
